docs/reference/run.md

```diff
@@ -623,6 +623,7 @@ container:
 | `--cpuset-mems=""`         | Memory nodes (MEMs) in which to allow execution (0-3, 0,1). Only effective on NUMA systems. |
```

Comment:
@Mashimiao did you push your changes? I see the note is still at the bottom (havent checked other changes)